### ISSUE TITLE
[BUG] create-playwright installs dependencies with npm without asking

### ISSUE THREAD

**Issue by ebeloded** [microsoft/create-playwright#15]
**Context:**
- create-playwright 1.17.110

**Describe the bug**

Adding playwright to an existing project (or create a new one) is very simple thanks to `create-playwright`. 

```shell
pnpm init playwright@latest
```

The problem is that it assumes `npm` and installs dependencies without asking first. When trying to add playwright to an existing project managed with `pnpm`, the init results in an error. 

It would be much better if `create-playwright` added necessary dependencies to `package.json`, but asked whether the developer wants to install them with `npm`.

**Comment by mattjennings** [comment]
Additionally, if the project has already been installed using pnpm, create-playwright may fail as it did for me:

```
6110 timing command:install Completed in 30678ms
6111 verbose stack TypeError: Cannot read properties of null (reading 'matches')
6111 verbose stack     at Link.matches (/Users/mattjennings/.nvm/versions/node/v16.13.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/node.js:1090:41)
6111 verbose stack     at Link.canDedupe (/Users/mattjennings/.nvm/versions/node/v16.13.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/node.js:1044:15)
6111 verbose stack     at PlaceDep.pruneDedupable (/Users/mattjennings/.nvm/versions/node/v16.13.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:465:14)
6111 verbose stack     at PlaceDep.placeInTree (/Users/mattjennings/.nvm/versions/node/v16.13.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:326:14)
6111 verbose stack     at PlaceDep.place (/Users/mattjennings/.nvm/versions/node/v16.13.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:214:10)
6111 verbose stack     at new PlaceDep (/Users/mattjennings/.nvm/versions/node/v16.13.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:71:10)
6111 verbose stack     at PlaceDep.placeInTree (/Users/mattjennings/.nvm/versions/node/v16.13.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:362:26)
6111 verbose stack     at PlaceDep.place (/Users/mattjennings/.nvm/versions/node/v16.13.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:214:10)
6111 verbose stack     at new PlaceDep (/Users/mattjennings/.nvm/versions/node/v16.13.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:71:10)
6111 verbose stack     at /Users/mattjennings/.nvm/versions/node/v16.13.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:964:31
6112 verbose cwd /Users/mattjennings/dev/svelte-pixi
6113 verbose Darwin 21.4.0
6114 verbose argv "/Users/mattjennings/.nvm/versions/node/v16.13.0/bin/node" "/Users/mattjennings/.nvm/versions/node/v16.13.0/bin/npm" "install" "--save-dev" "@playwright/experimental-ct-svelte"
6115 verbose node v16.13.0
6116 verbose npm  v8.5.4
6117 error Cannot read properties of null (reading 'matches')
```

The solution was to delete node_modules try again.

**Comment by mxschmitt** [comment]
pnpm supports was added in https://github.com/microsoft/create-playwright/pull/36 and gets released in a few days. Closing by that.

**Comment by KeionneDerousselle** [comment]
@mattjennings What was the workaround? I'm not sure I understand. I'm still seeing that error even after removing node_modules and running `pnpm create playwright` again.

**Comment by mattjennings** [comment]
@KeionneDerousselle sorry, I did word it confusingly! Here's what I did:

- delete node_modules
- `npm install`
- `npm init playwright`

now playwright should be setup properly, so you can go back to pnpm:

- delete node_modules
- delete package-lock.json
- `pnpm install`

**Comment by KeionneDerousselle** [comment]
> @KeionneDerousselle sorry, I did word it confusingly! Here's what I did:
> 
> * delete node_modules
> * `npm install`
> * `npm init playwright`
> 
> now playwright should be setup properly, so you can go back to pnpm:
> 
> * delete node_modules
> * delete package-lock.json
> * `pnpm install`

Worked like a charm!!!! Thank you!

**Comment by mxschmitt** [comment]
(I'll release a version with the fix next week, will be out of office until Tuesday and don't want to risk a broken deployment so didn't release it)

**Comment by mxschmitt** [comment]
Released as [v1.17.119](https://github.com/microsoft/create-playwright/releases/tag/v1.17.119).
